src/Stoa.ts

```diff
@@ -129,6 +134,12 @@ class Stoa extends WebService
         this.app.get("/wallet/transaction/overview/:hash", this.getWalletTransactionOverview.bind(this));
         this.app.get("/wallet/transactions/pending/:address", this.getWalletTransactionsPending.bind(this));
         this.app.get("/wallet/blocks/header", this.getWalletBlocksHeader.bind(this));
+        this.app.get("/latest-blocks", this.getLatestBlocks.bind(this));
+        this.app.get("/latest-transactions", this.getLatestTransactions.bind(this));
+        this.app.get("/block-summary", this.getBlockSummary.bind(this));
+        this.app.get("/block-enrollments", this.getBlockEnrollments.bind(this));
+        this.app.get("/block-transactions", this.getBlockTransactions.bind(this));
+        this.app.get("/boa-stats", this.getBOAStats.bind(this));
```

Comment:
`/boa-stats` Test code creation is required.

Reply:
I added the testcase.